scripts/main/lychee.js

```diff
@@ -75,6 +75,7 @@ let lychee = {
 	checkForUpdates: "1",
 	update_json: 0,
 	update_available: false,
+	new_photos_notification: true,
```

Comment:
I'm curious why this defaults to `true` given that it's supposed to be an opt-in feature?

Reply:
Yep, that should be false